lib/oli_web/templates/project/overview.html.eex

```diff
@@ -37,10 +38,32 @@
         <%= select f, :publisher_id, Enum.map(@publishers, &{&1.name, &1.id}), class: "form-control", required: true %>
       </div>
 
-      <%= submit "Save", class: "float-right btn btn-md btn-primary mt-2" %>
+    </div>
+  </div>
+
+  <div class="row py-5 border-bottom">
+  <div class="col-md-4">
+    <h4>Project Attributes</h4>
+    <div class="text-muted">
+      Project wide configuration, not all options may be relevant for all
+      subject areas.
+    </div>
+  </div>
+  <div class="col-md-8">
+
+    <%= inputs_for f, :attributes, fn fp -> %>
+      <div class="form-label-group mb-3">
+        <%= label fp, :learning_language, "Learning Language", class: "control-label" %>
+        <%= text_input fp, :learning_language, class: "form-control", required: false %>
```

Comment:
We'd want the same "<select>" construct here as well

Reply:
Got this in too

![image](https://user-images.githubusercontent.com/333265/193287948-fbb18d99-8910-45da-b83a-6d7e00574fe6.png)


Note: I did not change the schema to restrict the value to that list, do you think we should?